lib/surface/components/if.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Surface.Components.If do
+  @moduledoc """
+  Provides an alternative to the `:if` directive for wrapping multiple elements in an if expression.
+
+  ## Examples
+  ```
+  <If condition={{ @display_link }} />
```

Comment:
One thing we could try is to add the concept of "root prop" which would be similar to react's spread attribute:

```jsx
<Component {...props} />
```
But in our case, it would be treated just like any other property. Something like:

```elixir
prop condition, :boolean, required: true, root: true
...
prop each, :generator, required: true, root: true
```

Usage:

```jsx
<If {{ whatever == :done }}>
...
</If>

<For {{ item <- @items }}>
...
</For>
```

 I'm not sure yet how hard it would be to adjust the parser to handle that but it seems like a very nice addition, especially when start adding other stuff like `Case` and `Cond`. Naming is hard and I believe we'll end up using generic names like `expression` or `condition`. No need to implement that now, of course, but let me know if you like the idea so we can create an issue for it.

Reply:
I like the idea!